packages/gatsby/cache-dir/loader.js

```diff
@@ -38,6 +38,10 @@ function doFetch(url, method = `GET`) {
         resolve(req)
       }
     }
+    // Ensure responses are not cached. Usefull if page-data.json files are accidentally cached for example
+    if (noCache) {
+      req.setRequestHeader(`Cache-Control`, `no-store`)
```

Comment:
Let's move to no-cache so we still use disk-cache when etag and other validation methods are valid.

```suggestion
      req.setRequestHeader(`Cache-Control`, `no-cache`)
```